File: java/src/Behavioral/Command/DvdCommandNameStarsOn.java
package Behavioral.Command;


//DvdCommandNameStarsOn.java - one of two Concrete Commands

public class DvdCommandNameStarsOn extends CommandAbstract {
   private DvdName dvdName;  
    
   public DvdCommandNameStarsOn(DvdName dvdNameIn) {
       this.dvdName = dvdNameIn;
   }    
   public void execute() {
       this.dvdName.setNameStarsOn();
   } 
}

File: go/behavioral/command/DvdCommandNameStarsOn.go
package command


//DvdCommandNameStarsOn - one of two Concrete Commands

type DvdCommandNameStarsOn struct {
   dvdName *DvdName
}
    
func NewDvdCommandNameStarsOn(dvdNameIn *DvdName) *DvdCommandNameStarsOn {
	return &DvdCommandNameStarsOn{ dvdNameIn }
}    

func (this *DvdCommandNameStarsOn) execute () {
	 this.dvdName.SetNameStarsOn()
} 
